### ISSUE TITLE
Skip version 3, 4 and 5

### ISSUE THREAD

**Issue by pjasiun** [ckeditor/ckeditor5#652]
CKEditor 5 in version 4.6.2 will be very misleading. To avoid confusing between CKEditor and CKEditor 5 I believe we should skip versions 3 and 4.

At the same time, CKEditor 5, version 5.1 might be confused with CKEditor 5 version 1. This is why I think version 5 should be skipped as well. 

It means that after version 2 we will have directly version 6, and join to the modern versioning where some numbers are omitted ;)

**Comment by Reinmar** [comment]
This idea appeared in the past and it's definitely something worth considering. I'd actually consider skipping 2.x as well (FCKeditor had 2.x) and perhaps 1.x too in order to indicate that this is "newer" than any CKEditor 4.x.

So, the first stable version of CKEditor 5 would be 5.0.0 or 6.0.0 or even 10.0.0. Anything actually – just to make it clear that it's not 4.x.

cc @wwalc @fredck.

**Comment by Reinmar** [comment]
cc @wojtekidd

**Comment by wwalc** [comment]
I think I was for it as well in the past, but hasn't the decision already been made, because of semantic versioning, otherwise we'd not have 1.0.0-alpha (even if we went ahead with 1.0.0-alpha it still time before 1.0.0 stable to start with 5.0.0 I believe).

In any case, as both of you I see a problem in the future not only among users, being confused, but also with providing support, having confusing bug reports (mentioning steps to reproduce, version, but without screenshots or code samples that would give us some clue which product precisely it is) and so on. .

When thinking about CKEditor 5 versioning I think it's wise to think already what we will do with the next **generation** of the editor. Imagine that we're working on it. And we want to call it CKEditor 6. What to do then? Will we start the versioning from 26, because that was the highest version of CKEditor 5 that was released by then? Or from 40, to book some reasonable number of 14 versions for CKEditor 5 before it gets discontinued?
The problem may not be as big as it seems at the beginning. If CKEditor 5 reaches version e.g. 26 in a good few years, then by then the market share of CKEditor 4- will drop significantly. 
Which means, that the next generation (CKEditor 6) may restart its numbering from 1.0.0. We will differentiate CKEditor 5 and CKEditor 6 relatively easily (e.g. ver 26.0.0 ver 2.0.0 would tell us a lot)

So, to sum up. After my initial mixed feelings, I'm for starting with CKEditor 5 ver. 5.0.0 (and then have CKEditor 5 ver X.Y.Z according to sem ver).

**Comment by Reinmar** [comment]
We started from 10.0.0. Hurray! :D
